specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2021-04-01-preview/storagepool.json

```diff
@@ -0,0 +1,1803 @@
+    },
+    "DiskPoolUpdate": {
+      "type": "object",
+      "description": "Request payload for Update Disk Pool request.",
+      "required": [
+        "properties"
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Properties for Disk Pool update request.",
+          "$ref": "#/definitions/DiskPoolUpdateProperties"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "x-ms-mutability": [
+            "read",
+            "create",
+            "update"
+          ],
+          "description": "Resource tags."
+        }
+      }
+    },
+    "AvailabilityZone": {
+      "type": "string",
+      "description": "Availability zone."
+    },
+    "DiskPoolSku": {
+      "type": "object",
+      "required": [
+        "name"
+      ],
+      "description": "Sku for Disk Pools",
+      "properties": {
+        "name": {
+          "type": "string",
+          "name": "Name of the Sku"
+        },
+        "tier": {
+          "$ref": "#/definitions/DiskPoolTier",
+          "description": "Tier to use for the Disk Pool."
+        }
+      }
+    },
+    "DiskPoolProperties": {
+      "type": "object",
+      "description": "Disk Pool response properties.",
+      "required": [
+        "availabilityZones",
+        "subnetId",
+        "provisioningState",
+        "status"
+      ],
+      "properties": {
+        "provisioningState": {
+          "description": "State of the operation on the resource.",
+          "type": "string",
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "availabilityZones": {
+          "type": "array",
+          "description": "Logical zone for Disk Pool resource; example: [\"1\"].",
+          "items": {
+            "$ref": "#/definitions/AvailabilityZone"
+          }
+        },
+        "status": {
+          "type": "string",
+          "$ref": "#/definitions/OperationalStatus",
+          "description": "Operational status of the Disk Pool."
+        },
+        "disks": {
+          "type": "array",
+          "description": "List of Azure Managed Disks to attach to a Disk Pool.",
+          "items": {
+            "$ref": "#/definitions/Disk"
+          }
+        },
+        "subnetId": {
+          "type": "string",
+          "description": "Azure Resource ID of a Subnet for the Disk Pool.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "additionalCapabilities": {
+          "type": "array",
+          "description": "List of additional capabilities for Disk Pool.",
+          "items": {
+            "$ref": "#/definitions/AdditionalCapability"
+          }
+        },
+        "sku": {
+          "description": "Determines the SKU of the Disk Pool",
+          "type": "object",
+          "$ref": "#/definitions/DiskPoolSku"
+        }
```

Comment:
Is this needed? sku is already defined in DiskPool.

Reply:
Thanks for the correction!